Sarah_AlFadly/week_01/day_02/HW/if-else/SergeSays/main.js

```diff
@@ -0,0 +1,11 @@
+var message = "Can you help me ?";
+
+if (message.endsWith("?")) {
+    console.log("Sure");
+} else if (message === message.toLocaleUpperCase()) {
```

Comment:
Nice work here, interesting use of `toLocaleUpperCase()`. Remember, this is different than `toUpperCase()`. You can read more about the difference [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLocaleUpperCase).